vncviewer/Viewport.h

```diff
@@ -35,6 +35,10 @@ class CConn;
 class PlatformPixelBuffer;
 class Surface;
 
+// ExtendedMouseButtons
+const int MOUSE_FORWARD = 8;
+const int MOUSE_BACK = 9;
```

Comment:
Aren't these part of the new FLTK API? Seems strange to have them here in TigerVNC.

Reply:
Good point, I've removed these now.